src/main/java/org/terasology/potions/PotionCommonEffects.java

```diff
@@ -44,6 +44,7 @@
     public static final String FEATHER_FALL = "FEATHER_FALL";
     public static final String BRICK_FALL = "BRICK_FALL";
     public static final String MULTI_JUMP = "MULTI_JUMP";
+    public static final String NO_VISIBILITY="NO_VISIBILITY";//Awaiting API implementation for custom actions in Terasology for the NoVisibilityEffect to work.
```

Comment:
I cleaned this up just a tad (put the comment on its own line prefixed with a `TODO`)